.github/workflows/build.yml

```diff
@@ -408,12 +189,10 @@ jobs:
       # Version artifact based off of branch
       - name: Version Artifact
         run: |
-          version=${{ needs.setup.outputs.thisBranchFileName }}-SNAPSHOT
-          # Replace / with - in branch name
-          version=$(echo "$version" | sed 's/\//-/g')
+          version=${{ needs.setup.outputs.thisBranchName }}-SNAPSHOT
```

Comment:
Won't we face problems in branches with / in the name as the sed was removed?

Reply:
this is already taken care of in here -> https://github.com/liquibase/liquibase/pull/4780/files#diff-7314d0ebbd2e9537ae4889316745b4fd2fa43cb86275c9caae18a86ba228b642R79